src/fswAlgorithms/sensorInterfaces/TAMSensorData/_UnitTest/test_tamComm.py

```diff
@@ -92,7 +92,7 @@ def tamCommTestFunction(show_plots):
 
     # Initialize the test module configuration data
     dcm3, _ = np.linalg.qr(np.random.normal(0, 1, (3, 3)))
-    module.dcm_BS = dcm3.reshape(9, 1)
+    module.dcm_BS = unitTestSupport.flattenList(dcm3.reshape(9, 1).tolist())
```

Comment:
Why are we reshaping a dcm before sending it to the function? It should take care of that.

Reply:
Mm, I see what you mean. I could add npDCM2list() method to do this all in one.  I'll look into this later today.

Reply:
That would work, although the function you wrote already takes care of matrices too.

Reply:
Ha, you are right.  Just using `flattenList()` on the numpy DCM does yield the required list of DCM values.  I've made this change. good catch.